go/client/cmd_configure_nix.go

```diff
@@ -0,0 +1,309 @@
+	if toggleOn && toggleOff {
+		return fmt.Errorf("Cannot specify both --toggle-on and --toggle-off.")
+	}
+	if !toggleOn && !toggleOff {
+		return fmt.Errorf("Must specify either --toggle-on or --toggle-off.")
+	}
+	v.ToggleOn = toggleOn
+	return nil
+}
+
+func (v *CmdConfigureAutostart) Run() error {
+	err := install.ToggleAutostart(v.G(), v.ToggleOn, false)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func (v *CmdConfigureAutostart) GetUsage() libkb.Usage {
+	return libkb.Usage{
+		Config: true,
+		API:    true,
+	}
+}
+
+type CmdConfigureRedirector struct {
+	libkb.Contextified
+	Toggle   bool
+	ToggleOn bool
+	Status   bool
+}
+
+func NewCmdConfigureRedirector(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
+	cmd := &CmdConfigureRedirector{
+		Contextified: libkb.NewContextified(g),
+	}
+	return cli.Command{
+		Name: "redirector",
+		Usage: `
+
+Configure keybase-redirector settings.
+This option requires root privileges, and must use the root config file
+at /etc/keybase/config.json.
+
+The Keybase redirector allows every user to access KBFS files at /keybase, which
+will show different information depending on the requester.
+
+Enabling the root redirector will set suid root on /usr/bin/keybase-redirector,
+allowing any user to run it with root privileges. It is enabled by default.
+
+If the redirector is disabled, you can still access your files in
+the mount directory given by "$ keybase config get -d -b mountdir", which is
+owned by your user, but you won't be able to access your files at /keybase.
+
+More information is available at https://keybase.io/docs/kbfs/understanding_kbfs#mountpoints.
+
+Examples (prepend sudo, if not root):
+# keybase --use-root-config-file configure redirector --status
+# keybase --use-root-config-file configure redirector --toggle-on
+# keybase --use-root-config-file configure redirector --toggle-off`,
+		Flags: []cli.Flag{
+			cli.BoolFlag{
+				Name:  "status",
+				Usage: "Print whether the KBFS redirector is enabled or disabled.",
+			},
+			cli.BoolFlag{
+				Name:  "toggle-on",
+				Usage: "Toggle on the KBFS redirector.",
+			},
+			cli.BoolFlag{
+				Name:  "toggle-off",
+				Usage: "Toggle off the KBFS redirector.",
+			},
+		},
+		ArgumentHelp: "",
+		Action: func(c *cli.Context) {
+			cl.ChooseCommand(cmd, "redirector", c)
+			cl.SetForkCmd(libcmdline.NoFork)
+			cl.SetLogForward(libcmdline.LogForwardNone)
+		},
+	}
+}
+
+func xor3(a, b, c bool) (ret bool) {
+	ret = ret != a
+	ret = ret != b
+	ret = ret != c
+	return
+}
+
+func (v *CmdConfigureRedirector) ParseArgv(ctx *cli.Context) error {
+	toggleOn := ctx.Bool("toggle-on")
+	toggleOff := ctx.Bool("toggle-off")
+	status := ctx.Bool("status")
+	if !xor3(toggleOn, toggleOff, status) || (toggleOn && toggleOff && status) {
+		return fmt.Errorf("Must specify exactly one of --toggle-on, --toggle-off, --status.")
+	}
+	v.ToggleOn = toggleOn
+	v.Status = status
+	return nil
+}
+
+func (v *CmdConfigureRedirector) Run() error {
```

Comment:
Thanks for the review! I thought about that but thought that there might be some system where some non-root user has privileges to do these things. So it will fail trying to chmod if you can't.